x-pack/legacy/plugins/ml/server/models/data_recognizer/modules/siem_auditbeat_ecs/ml/suspicious_login_activity_ecs.json

```diff
@@ -0,0 +1,36 @@
+{
+  "job_type": "anomaly_detector",
+  "description": "SIEM Auditbeat: Detect unusually high increases in authentication attempts (beta)",
```

Comment:
let's change this to "SIEM Auditbeat: Detect unusually **high number of** authentication attempts (beta)"

Reply:
Done in d33a13304d9b8cb3b6555dc7b29711b7319b7f57